### PR TITLE
Fix lifecycle rules name

### PR DESCRIPTION
**Pull request by welansari**
Use proper lifecycle rule names defined here https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getBucketLifecycleConfiguration-property

Issue: BB-516

### CODE REVIEW

**Review by KillianG — APPROVED** [review]

File: tests/unit/lifecycle/LifecycleBucketProcessor.spec.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ describe('Lifecycle Bucket Processor', () => {
                         },
                         {
                             Status: 'Disabled',
-                            Transition: {
+                            Transitions: {
```
Comment (francoisferrand) [inline tests/unit/lifecycle/LifecycleBucketProcessor.spec.js]: Should this not be an array, as per AWS specs?

File: tests/unit/lifecycle/LifecycleBucketProcessor.spec.js [hunk 2]
```diff
@@ -104,7 +104,7 @@ describe('Lifecycle Bucket Processor', () => {
                         },
                         {
                             Status: 'Enabled',
-                            Transition: {
+                            Transitions: {
```
Comment (francoisferrand) [inline tests/unit/lifecycle/LifecycleBucketProcessor.spec.js]: Same here

File: extensions/lifecycle/util/rules.js [hunk 3]
```diff
@@ -202,14 +202,26 @@ function rulesToParams(versioningStatus, currentDate, bucketLCRules, bucketData,
...
         return rule.charAt(0).toUpperCase() + rule.slice(1);
     });
 }
 
+/**
+ * Check if ruleName is an expiration rule
+ * @param {string} ruleName lifecycle rule name
+ * @returns {bool} true if ruleName is an expiration rule
+ */
+function isExpirationRules(ruleName) {
```
Comment (francoisferrand) [inline extensions/lifecycle/util/rules.js]: nit: `isExpirationRule`

```suggestion
function isExpirationRule(ruleName) {
```

**Review by francoisferrand — APPROVED** [review]

**Pull request merged**